### PR TITLE
Release version 0.1.36

### PR DESCRIPTION
**Pull request by Anubhvv**
Release Version 0.1.36:

What's New:

```
Fluent V2:
	Fix/Enhancement:
	        1. Fixed the bug that caused multiline text to overlap in v2CCBActivity
	        2. Added trailing and leading accessories for the title in the section header, enhancing it's scope.
	        3. Resolved the issue of inconsistent background color in the side rail
	        4. Fixed the system dark and appearance light issues in the Fluent Demo App to ensure a consistent theme
	        5. Adding alignment parameter for pillbar so that user can customize the alignment of the pill buttons
```

### CODE REVIEW

File: FluentUI.Demo/src/main/assets/dogfood-release-notes.txt [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
 Fluent V2:
 	Fix/Enhancement:
-	        1. Fluent Icon RTL fix
-	Kotlin and Jetpack Compose Upgrade:
-		1.Jetpack compose BOM: 2023.06.01
-		2.Jetpack compose compiler version: 1.4.7 
-		3.Kotlin version: 1.8.21
\ No newline at end of file
+	        1. Fixed the bug that caused multiline text to overlap in v2CCBActivity
+	        2. Removed the developer id, email, and name from config.gradle and passed them as pipeline variables for better security and privacy
```
Comment (praveen970) [inline FluentUI.Demo/src/main/assets/dogfood-release-notes.txt:8]: I think this is not required since it is infra related.

Comment (Anubhvv) [reply FluentUI.Demo/src/main/assets/dogfood-release-notes.txt]: Done

File: config.gradle [hunk 2]
```diff
@@ -14,8 +14,8 @@
 project.ext.fluentui_calendar_versionid = '0.0.28'
 project.ext.fluentui_controls_versionid = '0.1.18'
 project.ext.fluentui_core_versionid = '0.1.27'
```
Comment (mishramayank1) [inline config.gradle:16]: update core version as this PR has changes in https://github.com/microsoft/fluentui-android/pull/513/files

File: config.gradle [hunk 3]
```diff
@@ -27,12 +27,12 @@ project.ext.fluentui_persona_versionid = '0.1.16'
 project.ext.fluentui_progress_versionid = '0.1.9'
 project.ext.fluentui_icons_versionid = '0.1.7'
 project.ext.fluentui_notification_versionid = '0.1.12'
-project.ext.FluentUI_versionid = '0.1.35'
+project.ext.FluentUI_versionid = '0.1.36'
 project.ext.fluentui_calendar_version_code = 28
 project.ext.fluentui_controls_version_code = 19
 project.ext.fluentui_core_version_code = 50
```
Comment (mishramayank1) [inline config.gradle:33]: update core version as this PR //github.com/microsoft/fluentui-android/pull/513/files has changes in core module

**Review by praveen970 — APPROVED** [review]

**Pull request merged**